docs/demos/first_steps.md

```diff
@@ -0,0 +1,299 @@
+---
+title: "OSM Demo: First Steps"
+description: "This demo of OSM is a walkthrough of setting up OSM, 2 pods, and applying an SMI policy to allow traffic between them."
+type: docs
+aliases: ["OSM Simple Demo"]
+weight: 2
+release: 0.8.0-rc.2
+---
+
+
+# Open Service Mesh Demo: First Steps
+
+
+> Note: This demo is specific to OSM v0.8.0-rc.2
+
+
+This document will walk you through the steps to:
+  - install Open Service Mesh (OSM) v0.8.0-rc.2
+  - install 2 sample apps: `bookbuyer` and `bookstore`
+  - apply [Service Mesh Interface (SMI)](https://smi-spec.io/) policies to control the traffic between these pods
+
+
+## Prerequisites
+This demo of OSM v0.8.0-rc.2 requires:
+  - a cluster running Kubernetes v1.15.0 or greater
+  - a workstation capable of executing [Bash](https://en.wikipedia.org/wiki/Bash_(Unix_shell)) scripts
+  - [The Kubernetes command-line tool](https://kubernetes.io/docs/tasks/tools/#kubectl) - `kubectl`
+
+
+## Download the OSM command-line tool
+
+The `osm` command-line tool contains everything needed to install and configure Open Service Mesh.
+The binary is available on the [OSM GitHub releases page](https://github.com/openservicemesh/osm/releases/).
+
+### For GNU/Linux or macOS
+Download and unzip the 64-bit [GNU/Linux](https://github.com/openservicemesh/osm/releases/download/v0.8.0-rc.2/osm-v0.8.0-rc.2-linux-amd64.tar.gz)
+or
+[macOS](https://github.com/openservicemesh/osm/releases/download/v0.8.0-rc.2/osm-v0.8.0-rc.2-darwin-amd64.tar.gz)
+binary of OSM v0.8.0-rc.2:
+```bash
+system=$(uname -s | tr '[:upper:]' '[:lower:]')
+release=v0.8.0-rc.2
+curl -L https://github.com/openservicemesh/osm/releases/download/${release}/osm-${release}-${system}-amd64.tar.gz | tar -vxzf -
+./${system}-amd64/osm version
+```
+
+### For Windows
+Download and unzip the Windows OSM v0.8.0-rc.2 binary via PowerShell:
+```powershell
+wget  https://github.com/openservicemesh/osm/releases/download/v0.8.0-rc.2/osm-v0.8.0-rc.2-windows-amd64.zip -o osm.zip
+unzip osm.zip
+.\windows-amd64\osm.exe version
+```
+
+### Compile from source
+The `osm` CLI can be compiled from source using [this guide](../../install/_index.md).
+
+
+## Installing OSM on Kubernetes
+
+With the `osm` binary downloaded and unzipped, we are ready to install Open Service Mesh on a Kubernetes cluster:
+
+```bash
+osm install
+```
+
+> Note: This document assumes you have already installed credentials for a Kubernetes cluster in ~/.kube/config and `kubectl cluster-info` executes successfully.
```

Comment:
Does the CLI require the kubeconfig file to be present? This should be documented as a prerequisite to running `osm install` if that's the case.

Reply:
The `~/.kube/config` file is by convention where the credentials for the kube client are. Without this file neither `kubectl` nor `osm` will work.
Example:
```
$ mv ~/.kube/config ~/.kube/config___
$ ./osm mesh list
Error: Could not list deployments Get "http://localhost:8080/apis/apps/v1/deployments?labelSelector=app%3Dosm-controller": dial tcp 127.0.0.1:8080: connect: connection refused


$ mv ~/.kube/config___ ~/.kube/config; osm mesh list

MESH NAME       NAMESPACE       CONTROLLER PODS
osm-many-many   osm-many-many   osm-controller-5b55f68568-8r46s
osm             osm-system      osm-controller-666f54645d-gqcn6
```